server/Makefile

```diff
@@ -0,0 +1,13 @@
+-include .env
+
+.PHONY: clean
+clean: ## clean build artifacts
+	rm -r ${BUILD_DIR} 2> /dev/null || :
+
+.PHONY: build
+build: clean ## build release file
+	(ls build 2> /dev/null || mkdir ${BUILD_DIR}) &&\
+	rustup target add ${BUILD_TARGET} &&\
+	cargo build --target=${BUILD_TARGET} --release 2> ${BUILD_DIR}/${BUILD_ERR_FILE} && \
```

Comment:
I haven't yet been able to get this to work, every time I run this, I get a flood of errors about libraries not being found. Currently trying to get this ironed out

Reply:
interesting, try a fresh clean and `rustup update`? m1 mac and intel macs worked for me after initially failing